### PR TITLE
Add lang and interface params in search requests

### PR DESCRIPTION
**Pull request by ger-benjamin**
c2cgeoportal ref : https://github.com/camptocamp/c2cgeoportal/issues/1840
Exemple: None yet [wip]

### CODE REVIEW

File: src/services/creategeojsonbloodhound.js [hunk 1]
```diff
@@ -53,29 +53,20 @@ ngeo.CreateGeoJSONBloodhound;
  * @param {ol.proj.Projection=} opt_dataProjection Data projection.
  * @param {BloodhoundOptions=} opt_options optional Bloodhound options. If
  *     undefined, the default Bloodhound config will be used.
- * @param {BloodhoundQueryOptions=} opt_query optional Bloodhound query options.
- *     Effective only if `remote`is not defined in `opt_options`.
+ * @param {BloodhoundRemoteOptions=} opt_remoteOptions optional Bloodhound
+ * remote options. Effective only if `remote`is not defined in `opt_options`.
```
Comment (sbrunner) [inline src/services/creategeojsonbloodhound.js]: `remote`is -> `remote` is

File: src/services/creategeojsonbloodhound.js [hunk 2]
```diff
@@ -53,29 +53,20 @@ ngeo.CreateGeoJSONBloodhound;
  * @param {ol.proj.Projection=} opt_dataProjection Data projection.
  * @param {BloodhoundOptions=} opt_options optional Bloodhound options. If
  *     undefined, the default Bloodhound config will be used.
- * @param {BloodhoundQueryOptions=} opt_query optional Bloodhound query options.
```
Comment (sbrunner) [inline src/services/creategeojsonbloodhound.js]: Shouldn't be remove in `typeahead.js`?

File: src/services/creategeojsonbloodhound.js (hunk 1 above)
Comment (sbrunner) [inline src/services/creategeojsonbloodhound.js]: with space

File: src/services/creategeojsonbloodhound.js [hunk 3]
```diff
@@ -53,29 +53,20 @@ ngeo.CreateGeoJSONBloodhound;
...
-        else {
-          var lastChar = settings.url.slice(-1);
-          if (lastChar != '&' && lastChar != '?') {
-            settings.url += settings.url.indexOf('?') < 0 ? '?' : '&';
-          }
-          settings.url += opt_query.param + '=' + query;
-        }
-        goog.object.extend(settings, opt_query);
+        settings.url = settings.url.replace('%QUERY', query);
+        settings.dataType = 'jsonp';
```
Comment (sbrunner) [inline src/services/creategeojsonbloodhound.js]: Not sure that's a good default in 2015...

File: src/services/creategeojsonbloodhound.js (hunk 2 above)
Comment (ger-benjamin) [inline src/services/creategeojsonbloodhound.js]: (Done)

File: src/services/creategeojsonbloodhound.js (hunk 1 above)
Comment (ger-benjamin) [inline src/services/creategeojsonbloodhound.js]: (Done)

**Comment by sbrunner** [comment]
Looks good for me :-)

**Comment by ger-benjamin** [comment]
I'll add tests later

**Pull request merged**